content/tutorials/kit-assembly.md

```diff
@@ -39,22 +46,23 @@ Make sure to read all these **before** you start assembly
 
+- DO NOT TOUCH THE MICRO SD CARD IN THE RASPBERRY PI
+
 ## How it call fits together
 
 {{% attachments title="Related files" /%}}
 
-1. Connect the Raspberry Pi to the Power Board using the Camcon to micro-usb cable; the small camcon end goes into one of the 5V rail sockets on the Power Board, the micro-usb end into the power input of the Raspberry Pi. (Once again, do not dismantle or rearrange this cable!)
+1. Connect the Raspberry Pi to the Power Board using the Camcon to micro-usb cable; the small camcon end goes into one of the 5V rail sockets on the Power Board, the micro-usb end into the power input of the Raspberry Pi. (Once again, do not dismantle or rearrange this cable!).
 2. Connect the USB hub to the Pi by plugging it into any one of its four USB sockets.
 3. Connect the Power Board to the Pi via one of the black micro-usb cables; the standard USB end goes into any USB socket on the Pi or connected USB hub, the micro-usb end into the Power Board.
-4. Connect the Motor Board to the Power Board by screwing the two large (7.5mm) camcons provided onto the opposite ends of a pair of wires, ensuring that positive connects to positive and ground to ground, and then plugging one end into the appropriate socket of the Motor Board and the other into a high power socket on the side of the Power Board.
+4. Connect the Motor Board to the Power Board by screwing the two large (7.5mm) camcons provided onto the opposite ends of a pair of wires, ensuring that positive connects to positive and ground to ground, and then plugging one end into the appropriate socket of the Motor Board (the connections labelled M0 or M1) and the other into a high power socket on the side of the Power Board ().
 5. Connect the Motor Board to the Pi by way of the other black micro-usb cable; the standard USB end goes into any USB socket on the Pi or connected USB hub, the micro-usb end goes into the Motor Board.
 6. The Arduino Servo Shield should already be connected into the top of the Arduino, but, if not, the pins on the bottom fit directly into the sockets with the same labels on the top of the Arduino.
-7. Connect the Arduino to the Pi by way of the beige USB-A to USB-B cable; the square-ish USB-B end goes into the appropriate metal-cased connector on the Arduino, the standard USB end goes into any free USB port on the Pi or connected USB hub. This will provide both power and communication to the Arduino.
+7. Connect the Arduino to the Pi by way of the beige USB-A to USB-B cable; the square-ish USB-B (this is the same cable the connects to a printer) end goes into the appropriate metal-cased connector on the Arduino, the standard USB end goes into any free USB port on the Pi or connected USB hub. This will provide both power and communication to the Arduino.
```

Comment:
The additional bracketed bit really isn't necessary. If people don't know what one looks like, maybe linking to an image in the original list would be better?